packages/ra-ui-materialui/src/layout/AppBar.js

```diff
@@ -70,41 +75,45 @@ const AppBar = ({
     toggleSidebar,
     ...rest
 }) => (
-    <MuiAppBar
-        className={classNames(classes.appBar, className)}
-        color="secondary"
-        position="absolute"
-        {...rest}
-    >
-        <Toolbar disableGutters variant="dense" className={classes.toolbar}>
-            <IconButton
-                color="inherit"
-                aria-label="open drawer"
-                onClick={toggleSidebar}
-                className={classNames(classes.menuButton)}
-            >
-                <MenuIcon
-                    classes={{
-                        root: open
-                            ? classes.menuButtonIconOpen
-                            : classes.menuButtonIconClosed,
-                    }}
-                />
-            </IconButton>
-            <Typography
-                variant="title"
-                color="inherit"
-                className={classes.title}
-            >
-                {typeof title === 'string' ? title : React.cloneElement(title)}
-            </Typography>
-            {logout &&
-                cloneElement(logout, {
-                    className: classes.logout,
-                })}
-        </Toolbar>
-        <LoadingIndicator className={classes.loadingIndicator} />
-    </MuiAppBar>
+    <Headroom className={classNames(classes.appBar)} style={styleHeadroom}>
```

Comment:
Breaking change here, you don't merge the `className` anymore

Reply:
I need advice on this one.

In fact, `className={className}` is on `<MuiAppBar />` components, maybe I have to put on `<Headroom` ?

Reply:
Convention is to apply `className` on the root element. This might be a breaking change though